### PR TITLE
ui: add multi-evidences support

### PR DESCRIPTION
**Pull request by simonvadee**

### CODE REVIEW

**Comment by such** [comment]
You should temporarly disable prettier in `.eslintrc` or configure it for this project. We don't want to change all single quotes to double quotes...

File: addon/components/map-explorer.js [hunk 1]
```diff
@@ -31,47 +31,51 @@ export default Ember.Component.extend({
...
+      );
       this.set(part + 'Showed', true);
     }
   },
 
   init() {
     this._super(...arguments);
     this.set('stateShowed', true);
-    this.set('evidence-component', 'bitcoin-component');
+    // this.set('evidence-component', 'evidences-component');
```
Comment (such) [inline addon/components/map-explorer.js]: no commented code ;)

File: addon/templates/components/evidences-component.hbs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="layout-row">
+
+  {{#each evidences as |evidence|}}
+    <div style="padding-right: 3rem">
+      {{#if (eq evidence.backend "TMPop") }} {{tmpop-evidence evidence=evidence}} {{else if (eq evidence.backend "bcbatch")}} {{bitcoin-evidence evidence=evidence}}
```
Comment (such) [inline addon/templates/components/evidences-component.hbs]: can we do something more dynamic like
```
{{#each evidences as |evidence|}}
  {{ component evidence.backend evidence=evidence }}
{{/each}}
```
?

Comment (simonvadee) [reply addon/templates/components/evidences-component.hbs]: I wasn't able to make it work before but now it's ok !

**Comment by simonvadee** [comment]
I added the eslint configuration for prettier

**Pull request merged**